From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jacob Champion <jacob.champion@ni.com>
Date: Tue, 2 Sep 2014 11:27:25 -0500
Subject: [PATCH] Don't use wincrypt.h on PharLap

The no-ssl version of libcurl uses Windows CryptoAPI for its MD5
implementation, but those entry points don't exist on PharLap. Use the
fallback instead.

Upstream-Status: Inappropriate [Pharlap specific]

Updated for curl 7.72.

Signed-off-by: Richard Tollerton <rich.tollerton@ni.com>

diff --git a/lib/md5.c b/lib/md5.c
index d21625f6b..aaabcc811 100644
--- a/lib/md5.c
+++ b/lib/md5.c
@@ -174,7 +174,7 @@ static void MD5_Final(unsigned char *digest, MD5_CTX *ctx)
   CC_MD5_Final(digest, ctx);
 }
 
-#elif defined(USE_WIN32_CRYPTO)
+#elif defined(_WIN32) && !defined(CURL_WINDOWS_APP) && !defined(PHARLAP)
 
 #include <wincrypt.h>
 #include "curl_memory.h"
-- 
2.28.0

